src/priceFeeder/laminar/feeder.ts

```diff
@@ -10,6 +10,9 @@ export default class LaminarFeeder extends SubstrateFeeder {
   oracleKeyFromListing({ symbol }: Listing): CurrencyId {
     return (currencyIds as any)[symbol];
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  async onNewPrices(prices: string[], listings: Listing[], nonce: number) { return Promise.resolve(); }
```

Comment:
some warnings here

Reply:
Fixed with a temp workaround `// eslint-disable-next-line`, will remove this method in the refactor PR.